frontend/packages/data-portal/app/components/Breadcrumbs.tsx

```diff
@@ -11,13 +10,23 @@ import {
 import { cns } from 'app/utils/cns'
 
 function encodeParams(params: [string, string | null][]): string {
-  const searchParams = new URLSearchParams(
-    (params.filter((kv) => isString(kv[1])) as string[][]).map((kv) =>
-      kv.map(encodeURIComponent),
```

Comment:
Wonder what would happen if we just removed this `encodeURIComponent()`. That might fix it?

Reply:
It does 👍

Reply:
I'm a bit hesitant that we will have weird characters that we want to escape though.  Thoughts @codemonkey800 ?

Reply:
I think `URLSearchParams` should handle the escaping automatically

Reply:
I thought so too, but I'm confused about how this works if it's escaped

Reply:
I think `useSearchParams()` unescapes stuff. Right now the code is double escaping ("Homo sapiens" turns into `organism=Homo%2520sapiens`)

Reply:
Ya, I also saw the double escape.  Sounds good - I'm going to go ahead and merge.